lib/commands/element.js

```diff
@@ -12,16 +12,19 @@ Object.assign(extensions, iosCommands.element);
 commands.getAttribute = async function (attribute, el) {
   el = util.unwrapElement(el);
   if (!this.isWebContext()) {
-    const value = await this.proxyCommand(`/element/${el}/attribute/${attribute}`, 'GET');
+    let value = await this.proxyCommand(`/element/${el}/attribute/${attribute}`, 'GET');
+    // Transform the result for the case when WDA returns an integer representation for a boolean value
+    if ([0, 1].indexOf(value) !== -1) {
```

Comment:
Here is the list of available element attributes: https://github.com/facebook/WebDriverAgent/blob/master/WebDriverAgentLib/Routing/FBElement.h

As you can see none of them returns numeric value